applications/luci-app-keepalived/htdocs/luci-static/resources/view/keepalived/vrrp_instance.js

```diff
@@ -0,0 +1,286 @@
+'use strict';
+'require view';
+'require form';
+'require uci';
+'require network';
+'require tools.widgets as widgets';
+
+return view.extend({
+	load: function() {
+		return Promise.all([
+			network.getDevices(),
+			uci.load('keepalived'),
+		]);
+	},
+
+	renderGeneralTab: function(s) {
+		var o, ipaddress;
+
+		o = s.taboption('general',form.Value, 'name', _('Name'));
+		o.rmempty = false;
+		o.optional = false;
+		o.placeholder = 'name';
```

Comment:
please Remove this placeholder text it does not help

Reply:
Updated